From 0272584a92a6fa8c1bbc0c4fa7f9f36fa6ce703d Mon Sep 17 00:00:00 2001
From: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
Date: Wed, 16 Feb 2022 20:03:45 +0300
Subject: [PATCH] arm64: dts: renesas: ulcb-kf: remove obsolete "kim" node

This node was used to add uart line discipline driver to use with
obsolete "btwilink" driver.

After removal of "btwilink", the line discipline is no longer needed.
The replacement, "wl18xx-st", uses in-kernel 'serdev' to communicate
with the serial port.

Signed-off-by: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/ulcb-kf.dtsi | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi b/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
index 457cd5050e3c..a4f91a994332 100644
--- a/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
@@ -130,16 +130,6 @@ vdd_cam_a_3: regulator@17 {
 		enable-active-high;
 	};
 
-	kim {
-		compatible = "kim";
-		shutdown-gpios = <&gpio_exp_74 3 GPIO_ACTIVE_HIGH>;
-		/* serial1 */
-		dev_name = "/dev/ttySC1";
-		flow_cntrl = <1>;
-		/* int div 8 hscif@26.6666656MHz */
-		baud_rate = <3333332>;
-	};
-
 	/delete-node/sound;
 
 	sound_card: sound@0 {
-- 
2.30.2

